### PR TITLE
[FEAT] [Navbar]

### PR DESCRIPTION
**Pull request by mferdiyanzah**
- add navbar
- add unit testing for navbar
- fix bug on navbar

peep

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,5 +5,9 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
   </head>
-  <body></body>
+  <body>
+    <nav>
```
Comment (mferdiyanzah) [inline index.html:9]: kodenya jelek cokkk

File: index.html [hunk 2]
```diff
@@ -5,5 +5,9 @@
...
+      <h1>Navbar cuyyy</h1>
```
Comment (zyvvva) [inline index.html:10]: code accepted

**Comment by ariframadhan1702** [comment]
bagus codenya

File: index.html [hunk 3]
```diff
@@ -5,5 +5,9 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
```
Comment (zyvvva) [inline index.html:6]: change title to belajar git kelompok 3

File: index.html (hunk 1 above)
Comment (ariframadhan1702) [reply index.html:9]: tes

**Review by zyvvva — APPROVED** [review]
accepted

**Review by ariframadhan1702 — APPROVED** [review]

**Comment by Aristaabdurrahman** [comment]
misi bang...

**Pull request merged**